GUI/MainWindow.xaml.cs

```diff
@@ -410,16 +410,16 @@ private void AddAFile(string draggedFilePath)
                             "please uninstall and reinstall MetaMorpheus", null));
                     }
 
-                    goto case ".mzml";
+                    goto case ".mzML";
 
-                case ".mzml":
+                case ".mzML":
```

Comment:
this is a problem because we are allowing file extensions to be named with variable casing (i.e., a file is valid input if it's named .mzml, .mzML, .MZML, etc etc. 

line 386 converts all of these variant cases to ".mzml" (with .ToLowerInvariant()) for easy parsing. please change this back to ".mzml"